provider/config.rb

```diff
@@ -202,6 +204,7 @@ def validate
       check_optional_property_list :style, Provider::Config::StyleException
       check_optional_property_list :changelog, Provider::Config::Changelog
       check_optional_property_list :functions, Provider::Config::Function
+      check_optional_property_list :overrides, Provider::AbstractOverride
```

Comment:
What's 'abstract' here mean? These are the overrides, so let's call them Overrides

Reply:
Abstract means that it is the "base" for TerraformOverride, ChefOverride, PuppetOverride and so on.

Reply:
Abstract usually means you cannot instantiate the object. Which in this case you can. So let's drop the Abstract prefix. Also using abstract usually means we lack naming contexts. Like:
Cat > Animal, Dog > Animal (not AbstractAnimal :))